src/scss/_variables.scss

```diff
@@ -0,0 +1,31 @@
+$blue:    #0d6efd;
```

Comment:
Most of these colors are already in Bootstrap. You don't need to add here again. You need to read this again https://content.noroff.dev/css-frameworks/bootstrap-with-sass.html

Reply:
https://github.com/NoroffFEU/css-frameworks-ca/pull/133

Reply:
i think i have fixed all of the issues now in the new branch

Reply:
it should have been fixed

Reply:
fixed all yesterday but my comment here didnt send